aea/configurations/schemas/aea-config_schema.json

```diff
@@ -59,15 +59,6 @@
         }
       }
     },
-    "ledger_apis": {
```

Comment:
we can also drop `path_to_contract_interface` in contract json now as it was only kept for backwards comp